paddle/fluid/inference/api/paddle_pass_builder.h

```diff
@@ -109,7 +109,10 @@ class CpuPassStrategy : public PassStrategy {
   CpuPassStrategy();
 
   explicit CpuPassStrategy(const CpuPassStrategy &other)
-      : PassStrategy(other.AllPasses()) {}
+      : PassStrategy(other.AllPasses()) {
+    use_gpu_ = other.use_gpu_;
+    use_mkldnn_ = other.use_mkldnn_;
```

Comment:
It don't need `use_mkldnn_quantizer_ = other.use_mkldnn_quantizer_`?

Reply:
Ah, true, I have missed that one.
Done.